src/aind_metadata_service/response_handler.py

```diff
@@ -87,116 +139,99 @@ def model_response(model: BaseModel) -> JSONResponse:
             model response from server
         """
         *_, validation_error = validate_model(model.__class__, model.__dict__)
-        model_json = jsonable_encoder(json.loads(model.json()))
         if validation_error:
-            response = JSONResponse(
-                status_code=StatusCodes.INVALID_DATA.value,
-                content=(
-                    {
-                        "message": f"Validation Errors: {validation_error}",
-                        "data": model_json,
-                    }
-                ),
-            )
+            status_code = StatusCodes.INVALID_DATA.value
         else:
-            response = JSONResponse(
-                status_code=StatusCodes.VALID_DATA.value,
-                content=(
-                    {
-                        "message": "Valid Model.",
-                        "data": model_json,
-                    }
-                ),
-            )
+            status_code = StatusCodes.VALID_DATA.value
+        response = (status_code, model)
         return response
 
     # flake8: noqa: C901
     @staticmethod
     def combine_procedure_responses(
-        lb_response: JSONResponse, sp_response: JSONResponse
-    ) -> JSONResponse:
+        lb_response: Tuple[int, Union[BaseModel, None]],
```

Comment:
Since these will always be Procedures Models, you shouldn't use the generic BaseModel class. I get warning about line 182: `lb_model.subject_procedures` because subject_procedures isn't an attribute of BaseModel.